wit/runtime_config.wit

```diff
@@ -1,16 +1,21 @@
-package wasi:config@0.1.0;
+package wasi:runtime-config@0.1.0;
```

Comment:
When I submitted the PR, we decided we wanted to keep `wasi:config` and just have the interface be called `runtime`. Is there a reason we want to change it now?

Reply:
No, I will revert it back